### PR TITLE
treadmill

### PR DESCRIPTION
**Pull request by valeIT**

### CODE REVIEW

File: _posts/2019-10-31-treadmill-desk.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+type: post
+published: true
+status: publish
+categories: [Tech]
+image:
+image2:
+author: Valentino Urbano
+---
+
+A few days ago I've written about how using a [standing desk][1] is good for your health and advised everyone to do it.
```
Comment (MsSlacker) [inline _posts/2019-10-31-treadmill-desk.md]: Try it instead do it? Seems less harsh.

File: _posts/2019-10-31-treadmill-desk.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+What I haven't mentioned is, that's not the whole story. We also have a treadmill desk at the office, and as for the standing desk it is rarely used. Which means that I can basically use it whenever I want.
```
Comment (MsSlacker) [inline _posts/2019-10-31-treadmill-desk.md]: and as for the standing desk it is rarely used - > and it's also rarely used

File: _posts/2019-10-31-treadmill-desk.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+You should get familiar with the standing desk first and be comfortable working on it before trying the treadmill desk.
+
+## Benefits
+
+I never thought I could work while walking on it, but it goes rather slowly and for most things, once you get the hang of it, it is totally fine. You're not going to be more productive than if you're not walking, but the loss of productivity is minimal and the health benefits are huge.
```
Comment (MsSlacker) [inline _posts/2019-10-31-treadmill-desk.md]: You're not going to be more productive than if you're not walking, but the loss of productivity is minimal and the health benefits are huge. - > The loss of productivity is minimal and the health benefits are huge.

File: _posts/2019-10-31-treadmill-desk.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+For those reasons I'm trying to get at least an hour of it every 2 days. I will try to switch it to every day but for now every two days looks best.
```
Comment (MsSlacker) [inline _posts/2019-10-31-treadmill-desk.md]: I will try - > my goal is to

Comment (MsSlacker) [reply _posts/2019-10-31-treadmill-desk.md]: Looks best - > works best for me

File: _posts/2019-10-31-treadmill-desk.md [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+
+## How To Start
+
+For the first few times assume that your productivity will be cut by half. It may not be, but it is different for different people.
+
+You should pick a day when you won't be doing any intensive task (like programming).
+What can you do on it? Mails, scheduling, light UI work, code review.
```
Comment (MsSlacker) [inline _posts/2019-10-31-treadmill-desk.md:32]: Missing an "enter"? (space)

**Review by MsSlacker — CHANGES_REQUESTED** [review]
Did you notice any changes? Besides being better at it and being able to do more?

**Comment by valeIT** [comment]
> Did you notice any changes? Besides being better at it and being able to do more?

Not really, mostly I can work while doing it no problem unless it is a particularly hard problem.

File: _posts/2019-10-31-treadmill-desk.md (hunk 1 above)
Comment (valeIT) [reply _posts/2019-10-31-treadmill-desk.md]: sure

File: _posts/2019-10-31-treadmill-desk.md (hunk 3 above)
Comment (valeIT) [reply _posts/2019-10-31-treadmill-desk.md]: Like that sounds that is only good. Sounds also be pointed out that the loss is there.

File: _posts/2019-10-31-treadmill-desk.md (hunk 5 above)
Comment (valeIT) [reply _posts/2019-10-31-treadmill-desk.md:32]: yup

**Review by MsSlacker — APPROVED** [review]

**Pull request merged**